### ISSUE TITLE
Re-adding custom chat

### ISSUE THREAD

**Issue by RvawcTheDev** [Epix-Incorporated/Adonis#1576]
### What part of Adonis is this related to?

Other

### What are you suggesting?

Hey there this is a request for the custom chat to come back. The reason for this is I used it in one of my SCPF Games on Roblox and when it got removed in the last update to made my game unplayable for Xbox users. So I was wondering if it could be added back as I would say multiple people will love to have it back. -RvawcTheDev

**Comment by WalkerOfBacon** [comment]
Its against roblox TOS to let console users use chat

**Comment by RvawcTheDev** [comment]
> Its against roblox TOS to let console users use chat

Yes but its still something all the console players want I know its against TOS but it's been a thing for years and they did nothing about it plus if the console players will really start to storm Github to get it back up soon or spam the creators.

**Comment by Dimenpsyonal** [comment]
we won't add something that's against the TOS

**Comment by javajuicer** [comment]
use at your own risk
https://github.com/Epix-Incorporated/Adonis-Plugins/blob/master/Server/Server-AdonisChatAddition.rbxmx

**Comment by GalacticInspired** [comment]
agaisnt tos
